### PR TITLE
Responsive styling

### PR DESCRIPTION
**Pull request by KongLam03**
Media Queries. Naming conventions are difficult!!

### CODE REVIEW

**Comment by quizzy** [comment]
I pointed the base branch of this PR to `DRY-functions` branch as I believe that is from where you branched off. This makes it easier to read the changes.

File: src/components/TabBarComponent.js [hunk 1]
```diff
@@ -47,14 +48,28 @@ class TabBarComponent extends React.Component {
...
+		favouritesText: {
+			fontSize: 11
+		},
+		nearbyText: {
+			fontSize: 12
+		},
+		recentText: {
+			fontSize: 12
+		},
+		mapText: {
```
Comment (quizzy) [inline src/components/TabBarComponent.js]: Don't you need to supply default font sizes too?

File: src/components/TabBarComponent.js [hunk 2]
```diff
@@ -47,14 +48,28 @@ class TabBarComponent extends React.Component {
 	}
 }
 
-let styles = StyleSheet.create({
+let styles = EStyleSheet.create({
```
Comment (quizzy) [inline src/components/TabBarComponent.js]: `const` over `let` since it doesn't change.

File: src/components/liveBusTimes/LiveTimesView.js [hunk 3]
```diff
@@ -25,10 +26,10 @@ export const LiveTimesView = ({bustime, refresh, filteredBusTime, busTimes}) =>
 										<Text>{e.lineName}</Text>
 									</View>
 									<View style={styles.destinationName}>
-										<Text>{e.destinationName}</Text>
+										<Text style={styles.destinationText}>{e.destinationName}</Text>
 									</View>
 									<View style={styles.timeInMins}>
-										<Text>{timeInMins}</Text>
+										<Text style={{color: timeInMins == 'Due' ? 'red' : 'black', fontWeight: timeInMins == 'Due' ? '800' : null}}>{timeInMins}</Text>
```
Comment (quizzy) [inline src/components/liveBusTimes/LiveTimesView.js]: Nice

File: src/components/TabBarComponent.js (hunk 1 above)
Comment (KongLam03) [reply src/components/TabBarComponent.js]: Oh yeah!! LOL.

**Pull request closed without merging**